### ISSUE TITLE
Program stuck at return sess.run([self.logits],  {self.encoder_inputs : source_batch_major, self.source_lengths : source_seq_lengths}) in sample function in chatbot.py

### ISSUE THREAD

**Issue by mittal-pm** [inikdom/neural-chatbot#28]
![IMG-20200326-WA0000](https://user-images.githubusercontent.com/34447717/77673153-ceefbf80-6faf-11ea-9966-3f98a0873f5c.jpeg)
